packages/design-system/src/interfaces/forms.d.ts

```diff
@@ -181,6 +181,13 @@ export interface ISearchBarProps extends ISearchBarHtmlProps {
   onExpand?(): void;
 }
 
+export interface IIconProps {
```

Comment:
please extend some html interface to make use of this rest operator you use in the component